### DISCUSSION TITLE
Add many to many relation to query result

### DISCUSSION THREAD

**Discussion by alucardu** [prisma/prisma#14795] [Q&A]
I have a query that returns a list of movies:

```
const returnMoviesFromUser = async (args, req, res) => {
  if (req.userId) {
    const user = await prisma.user.findUnique({
      where: {id: args.userId || req.userId},
      select: {
        movies: true,
      },
    });
    return user.movies || [];
  }
};
```
A movie can have many ratings:
```
model Movie {
  ...
  movieRating    MovieRating[]
}
```
This is how I find a movie rating belonging to a movie:
```
returnMovieRating: async (_, args, {req}) => {
  return prisma.movieRating.findFirst({
    where: {
      userId: args.userId,
      movieId: args.movieId,
    },
  });
},
```
Would it be possible to return the movie rating with the movie object in the `returnMoviesFromUser` query?

**Comment by l2D** [comment]
I think you could try these:

```js
const returnMoviesFromUser = async (args, req, res) => {
  if (req.userId) {
    const user = await prisma.user.findUnique({
      where: {id: args.userId || req.userId},
      include: {
        movies: {
          include: {
             movieRating: true
          }
        },
      },
    });
    return user.movies || [];
  }
};
```

```js
const returnMoviesFromUser = async (args, req, res) => {
  if (req.userId) {
    const user = await prisma.user.findUnique({
      where: {id: args.userId || req.userId},
      include: {
        movies: {
          include: {
            movieRating: {
              select: {
                 rating: true,
                 updated_at: true
              }
            }
          }
        },
      },
    });
    return user.movies || [];
  }
};
```

**Reply by vladi-stevanovic** [reply]
👋 Hello @alucardu I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
